invenio_grobid/api.py

```diff
@@ -33,16 +33,13 @@
 from .errors import GrobidRequestError
 
 
-def process_pdf(pdf_file, stream=False):
+def process_pdf(pdf_file):
```

Comment:
Shall we indicate that it expects a stream object? E.g. `process_pdf_stream` or in the docstring?